tests/test_cmd_init.py

```diff
@@ -114,14 +113,23 @@ def test_existing_db_error(self):
         self.assertEqual(code1, CMD_SUCCESS)
 
         code2 = self.cmd.run(self.name)
-        self.assertEqual(code2, CODE_DATABASE_ERROR)
+        self.assertEqual(code2, CODE_DATABASE_EXISTS)
 
         # Context added to catch deprecation warnings raised on Python 3
         with warnings.catch_warnings(record=True):
             output = sys.stderr.getvalue().strip()
             self.assertRegexpMatches(output,
                                      DB_EXISTS_ERROR % {'database' : self.name})
 
+    def test_existing_db_reuse(self):
+        """Check if it returns an error when tries to create the registry twice"""
```

Comment:
The description of this test is wrong. Calling to run doesn't return an error in any of the cases.